### ISSUE TITLE
Describe relationship to Tide

### ISSUE THREAD

**Issue by jglick** [jenkins-x/lighthouse#51]
https://github.com/jenkins-x/lighthouse/#comparisons-to-prow does not mention Tide, one of the most significant Prow plugins. In particular, its feature of batch-merging PRs whose head commit has already been vetted is a major advantage. It is not obvious what, if any, equivalent Lighthouse offers. (As I recall, Tide runs as a background service rather than as a Hook plugin, so it would not fit into the Lighthouse model unless it were redesigned to react to events.) This should be discussed in the README.

**Comment by jstrachan** [comment]
@jglick agreed - we definitely need something tide-like for sure! The immediate needs are low footprint + multi-git provider so we've punted so far on tide but its important to add this soon.

Maybe we can add Tide as a kind of App which if enabled runs a tide step in the meta pipeline/pipeline (or separate pipeline) which decides what additional pipelines (if any) to run in parallel. e.g. we could run a pipeline to decide if we should run a batch merge whenever we trigger a pipeline or it complete or something?

**Comment by jglick** [comment]
To be clear, I filed this just as a docs task: adjust README to mention that Tide exists, what it is useful for, and the fact that Lighthouse does not currently offer a replacement.

**Comment by ccojocar** [comment]
merge automation (e.g. something like tide) is not a priority for phase 1 but we added this requirement for later phases.  

I am going to close this issue for now to keep the things clean and focused. Thanks for reporting this.
